foulkon/worker.go

```diff
@@ -310,9 +310,15 @@ func getDefaultValue(config *toml.Tree, key string, def string) string {
 
 // Check variables in TOML file.
 // If the value of a key is '${SOME_KEY}', we will search the value in the OS ENV vars
+// If the value of a key is not a string, but an number, it converts it to a string,
```

Comment:
Hey, "but an number" should be "but a number". Am i right?. Anyway your PR looks ok for me. Thx :smile:

Reply:
OH shoot. Yeah. Fix incoming.